src/main/java/com/osiris/autoplug/client/SystemChecker.java

```diff
@@ -70,6 +73,12 @@ public void addShutDownHook() {
                 AL.warn("Error during shutdown, related to stopping the server!", e);
             }
 
+            try {
+                SSHManager.stop();
```

Comment:
why here?

Reply:
yeah idk, closing the JRE closes ssh anyway. removed

Reply:
oh didn't see it was inside the shutdownhook function, my bad, then it makes sense having that there to ensure gracefully closing it.